packages/sdk/test/cross-chain-messenger.spec.ts

```diff
@@ -25,6 +25,22 @@ describe('CrossChainMessenger', () => {
   let l2Signer: any
   before(async () => {
     ;[l1Signer, l2Signer] = await ethers.getSigners()
+    // ethers.getSigners() is out of date - recent versions of geth
+    // include a balance check as part of the estimateGas() call.
```

Comment:
One issue with this is optimism contracts themselves expect you to estimate gas with Address(1) to correctly estimate https://github.com/ethereum-optimism/optimism/blob/5bafddac62e41a0b6744d0aac5a0a178b11788be/packages/contracts-bedrock/src/L1/OptimismPortal2.sol#L381

Reply:
that is okay since the function is non-payable